### DISCUSSION TITLE
My Recent Activity - A simple, chronological list of recent changes to my repos

### DISCUSSION THREAD

**Discussion by joehanna** [community/community#15301] [Other Feature Feedback, Questions, & Ideas]
Can we please just have a log of changes to all of my collaborating repos?  I just want to see what my team has done in terms of:-
* commits
* issue changes
* PR submissions
* Changes to Wiki

A simple, chronological list of recent changes, in descending time order - PLEASE!

Thanks

**Comment by jaypavasiya** [comment]
Agree！
